src/web/views/Components/SearchAutocomplete/props-data.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable */
+export default [
+  [ "Props", "Type", "Default value", "Description" ],
+  [
+    "items",
+    "Array",
+    "() => []",
+    "Use this props to pass your items to the component"
```

Comment:
As the format/type of what is contained in this array is well defined, I think it should be described in the doc.